.github/workflows/main.yaml

```diff
@@ -297,3 +297,39 @@ jobs:
           fetch-depth: 0
 
       - run: make image
+
+
+  scan_vulnerabilities:
+    name: "[Trivy] Scan for vulnerabilities"
+    runs-on: ubuntu-22.04
+    timeout-minutes: 30
+    permissions:
+      contents: read
+    steps:
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
+
+      - name: Download trivy
+        run: |
+          pushd $(mktemp -d)
+          wget https://github.com/aquasecurity/trivy/releases/download/v${{ env.TRIVY_VERSION }}/trivy_${{ env.TRIVY_VERSION }}_Linux-64bit.tar.gz
+          tar zxvf trivy_${{ env.TRIVY_VERSION }}_Linux-64bit.tar.gz
+          echo "$(pwd)" >> $GITHUB_PATH
+        env:
+          TRIVY_VERSION: "0.32.1"
+
+      - name: Run trivy on git repository
+        run: |
+          trivy fs --format table --ignore-unfixed --skip-dirs website --security-checks vuln .
+
+      - name: Build docker images
+        run: make image
+
+      - name: Run trivy on images
+        run: |
+          images=`docker images | grep kubeedge | awk '{print $1":"$2}'`
+          for img in ${images[@]}; do
```

Comment:
Please use the `strategy.matrix` to run the jobs concurrently.
FYI: https://github.com/kubeedge/kubeedge/blob/master/.github/workflows/release.yml#L15